website/pages/docs/recipes/sources/aws.md

```diff
@@ -9,5 +9,22 @@ spec:
   path: cloudquery/aws
   version: "v4.7.5" # latest version of aws plugin
   tables: ["*"]
+
+  # Comment out any of the following tables if you want to sync them
+  # unless otherwise indicated they are configuration parameters rather than configured resources
+  skip_tables:
+    - aws_ec2_vpc_endpoint_services # this resource includes services that are available from AWS as well as other AWS Accounts
+    - aws_docdb_cluster_parameter_groups
+    - aws_docdb_engine_versions
+    - aws_ec2_instance_types
+    - aws_elasticache_engine_versions
+    - aws_elasticache_parameter_groups
+    - aws_elasticache_reserved_cache_nodes_offerings
+    - aws_elasticache_service_updates
+    - aws_neptune_cluster_parameter_groups
```

Comment:
add `aws_lambda_runtimes` as well?

Reply:
I agree that it is a configuration rather than a resource, but it also doesn't require an API call